atom/browser/api/atom_api_app.cc

```diff
@@ -380,7 +380,7 @@ void OnClientCertificateSelected(
     mate::Arguments* args) {
   mate::Dictionary cert_data;
   if (!args->GetNext(&cert_data)) {
-    args->ThrowError();
```

Comment:
So if you call the callback with an empty object, string, number, etc. now it will continue the request where previously it would error out.

Perhaps it would be better to explicitly check for the argument being null/undefined and still throw an error when an invalid parameter was specified to prevent accidental continuing.

What do you think?